app/views/wikis/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-   <div class="col-md-8">
+   <div class="col-md-10">
      <h2 class="text-center" style="margin: 20px;"><u>My wikis</u></h2>
      <% if policy_scope(@wikis).count > 0 %>
```

Comment:
@wonderwoman13 Nothing specific to clarify,it was a copy paste :).I removed the policy scope from the  index html.